tests/test_other.py

```diff
@@ -8343,6 +8343,9 @@ def test_autotools_shared_check(self):
   def test_ioctl_window_size(self):
       self.do_other_test('test_ioctl_window_size.cpp')
 
+  def test_sys_ioctl(self):
+    self.do_other_test('test_ioctl.c', emcc_args=['-sFORCE_FILESYSTEM', '-sEXIT_RUNTIME'])
```

Comment:
```suggestion
    self.do_other_test('test_ioctl.c')
```
The test passes without these, I'm guessing?

Reply:
The ioctl syscall needs file system, https://github.com/emscripten-core/emscripten/blob/57c61a5b8174d35fd2bb2386c5fc03f4daeb51d9/src/library_syscall.js#L203-L210

Reply:
I see. In that case please add a comment clarifying that.

What about the other flag, exit runtime?